crates/devolutions-log/Cargo.toml

```diff
@@ -0,0 +1,28 @@
+[package]
+name = "devolutions-log"
```

Comment:
Made a common logging library to reduce code duplication, it is almost identical between Gateway & Agent